src/components/cylc/cylcObject/Menu.vue

```diff
@@ -204,10 +204,7 @@ export default {
       const shortList = this.primaryMutations
       if (!this.expanded && shortList.length) {
         return this.mutations
-          // filter for shortlisted mutations
-          .filter(x => shortList.includes(x.mutation.name))
-          // filter out mutations which aren't relevant to the workflow state
-          .filter(x => !this.isDisabled(x.mutation, true))
+          .filter(x => shortList.includes(x.mutation.name) && !this.isDisabled(x.mutation, true))
```

Comment:
JS Noob question: Is this preferred because its faster, easier to read (I might disagree), or better in some other way?

Reply:
It's going to be faster because it is iterating over the array once instead of twice